admin/pkg/adminenv/admin_env.go

```diff
@@ -0,0 +1,29 @@
+package adminenv
```

Comment:
Can we move this package to `cli/pkg`? Since it relates to CLI configuration. (The admin service is, in theory, generic.)